modules/metastore/modules/metastore_search/config/install/search_api.index.dkan.yml

```diff
@@ -26,6 +26,11 @@ field_settings:
     datasource_id: dkan_dataset
     property_path: modified
     type: string
+  nid:
```

Comment:
I think we need an update hook to do the config import task automatically as well, you shouldn't have to run `drush cim` when you enable a module.